sample_app/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: ..
   specs:
-    aws-sdk-rails (3.12.0)
+    aws-sdk-rails (4.0.0)
```

Comment:
I think we can delete Gemfile.lock and add it to gitignore. I can't remember with this repo if this is necessary.